app/controllers/places_controller.rb

```diff
@@ -2,6 +2,12 @@ class PlacesController < ApplicationController
   def index
   end
 
+  def feed
```

Comment:
See https://github.com/mainelabs/techlist/pull/59#issuecomment-100263040. Could be moved to a `respond_to` in `api/v1/places#index`.